### ISSUE TITLE
jc.removeJobs not working

### ISSUE THREAD

**Issue by Nevtep** [vsivsi/meteor-job#10]
I'm trying to run a cleanup job to remove old jobs, code goes like this:

```
cleanupJobQueue = workers.processJobs('cleanup', function(job, cb) {
    console.log("Running cleanup job")
    var current, ids;
    current = new Date();
    current.setMinutes(current.getMinutes() - 60);
    console.log("cleanup - set current to:", current)
    ids = workers.find({
      status: {
        $in: Job.jobStatusRemovable
      },
      updated: {
        $lt: current
      }
    }, {
      fields: {
        _id: 1
      }
    }).map(function(d) {
      return d._id;
    });
    console.log("cleanup - found ids:", ids.length)

    workers.removeJobs(ids, {}, function(err, result) {
      if(!result)
        console.log("cleanup - error:", err);
      else
        console.log("cleanup - success");
      if (ids.length > 0) {
        console.log("Removed " + ids.length + " old jobs");
      } else {
        console.log("Nothing to remove");
      }
      job.done();
      cb();
    });
  });
```

The jobs are not being removed, this is the log I get:

 Running cleanup job 
 cleanup - set current to: Tue Apr 12 2016 04:48:48 GMT+0000 (UTC) 
 cleanup - found ids: 8239 

Nothing else beyond this line, no error, no exception thrown, no Nothing to remove, no messages at all and job is still "running on the db" (looks like the callback is bit being called) what could be causing this?

thank you for your answers

**Comment by vsivsi** [comment]
Hi, I noticed that you have filed this against the meteor-job project and not meteor-job-collection, so I am assuming that you are running the code above in a vanilla node.js process, and not in the Meteor environment...

Two things to try... One is that you should turn on logging on the server you are connecting to, so that you can see everything happening on that end. It's very helpful for diagnosing these types of issues. See: https://github.com/vsivsi/meteor-job-collection#jcsetlogstreamwritestream---server-only

The other thing you don't discuss is how the server is configured with regard to allow/deny rules and authentication. For any client, including a node.js client, there must be an `allow` rule in place that gives permission for a client to remove jobs. So for example, your node.js process should be authenticated as a userId that belongs to the `'manager'` or `'admin'` group. 

Beyond those suggestions, I'm sure you can appreciate the difficulty of remotely debugging an issue like this via github messages. If you can't figure it out, the best way to proceed would be to create a minimal github project that clearly reproduces the issue and link to it here. Then I'll be happy to look at the code and figure out what is going on.

BTW, I like your "glider" avatar!

**Comment by vsivsi** [comment]
Fixed in 1.3.3, just published to npm
